LFGBulletinBoard/dungeons/classic.lua

```diff
@@ -128,36 +134,8 @@ for key, id in pairs(LFGActivityIDs) do
     idToDungeonKey[id] = key
 end
 
---- Any info that needs to be overridden for a specific dungeon should be done here.
+--- Any info that needs to be overridden/spoofed for a specific dungeon should be done here.
 local infoOverrides = {
-    -- BFD
-    BFD = isSoD and {
-        typeID = DungeonType.Raid,
-        minLevel = 25,
-        maxLevel = 40,
-        size = 10,
-    },
-    -- Gnomer
-    GNO = isSoD and {
-        typeID = DungeonType.Raid,
-        minLevel = 40,
-        maxLevel = 50,
-        size = 10,
-    },
-    -- Sunken Temple
-    ST = isSoD and {
-        typeID = DungeonType.Raid,
-        minLevel = 50,
-        maxLevel = 60,
-        size = 20,
-    },
-    -- Demon Fall Canyon (completely spoofed for SoD since its not added into the LFGDungeon db2 table)
-    DFC = isSoD and {
-        name = GetRealZoneText(2784),
-        minLevel = 60,
-        maxLevel = 60,
-        typeID = DungeonType.Dungeon,
-    },
     WB = isSoD and {
```

Comment:
the `C_LFGList.GetActivityInfoTable` now correctly reports this data when using the SoD IDs